File: problem/C++/TwoSum.cpp
#include <map>

class Solution {

public:
    vector<int> twoSum(vector<int>& nums, int target) {
        unordered_map<int,int> conv;
        int j = 0;
        for(auto i : nums){
            if(conv[target - i])
                return {conv[target-i]-1,j};
            else
                conv[i] = ++j;
        }
        return {};
    }
};

File: problem/bry/Kotlin/twoSum.kt
class Solution {
    fun twoSum(nums: IntArray, target: Int): IntArray {
        val conv = HashMap<Int, Int>()
        nums.forEachIndexed{index, int ->
            conv[int]?.let{return intArrayOf(it, index)}
            conv[target - int] = index
        }
        return intArrayOf()
    }
}